pom.xml

```diff
@@ -49,13 +49,13 @@
         <lombok.version>1.18.20</lombok.version>
         <lombok-maven-plugin.version>1.18.16.0</lombok-maven-plugin.version>
         <delombok.output>target/generated-sources-for-javadocs</delombok.output>
-        <kotlin.version>1.5.10</kotlin.version>
-        <slf4j.version>1.7.30</slf4j.version>
-        <aws.s3.version>1.11.1031</aws.s3.version>
+        <kotlin.version>1.5.20</kotlin.version>
+        <slf4j.version>1.7.31</slf4j.version>
+        <aws.s3.version>1.12.14</aws.s3.version>
```

Comment:
I will verify if there is no breaking change, just in case.

Reply:
I didn't detect any issues. The main changes by the upgrade is the underlying jackson library upgrade: https://aws.amazon.com/blogs/developer/aws-sdk-for-java-version-1-12/